v2_1/ws/rest/docs/4_4_data_queries.md

```diff
@@ -38,11 +38,11 @@ The following query parameters are supported:
 
 Parameter | Type | Description
 --- | --- | ---
-c | Map | Filter data by component value. For example, if a structure defines a frequency dimension (FREQ) and the code A (Annual) is an allowed value for that dimension, the following can be used to retrieve annual data: `c[FREQ]=A`. The same applies to attributes (e.g. `c[CONF_STATUS]=F`) and measures. Multiple values are supported, using a comma (`,`) as separator: `C[FREQ]=A,M`. In case of attributes that support multiple values, the plus (`+`) can be used to list all values that an attribute must have. For example, to indicate that ATTR1 must either be A or B AND M, use the following: `C[ATTR1]=A,B+M`. Operators may be used too (see table with operators below). This parameter can be used in addition, or instead of, the `key` path parameter. 
+c | Map | Filter data by component value. For example, if a structure defines a frequency dimension (FREQ) and the code A (Annual) is an allowed value for that dimension, the following can be used to retrieve annual data: `c[FREQ]=A`. The same applies to attributes (e.g. `c[CONF_STATUS]=F`) and measures. Multiple values are supported, using a comma (`,`) as separator: `C[FREQ]=A,M`. In case of attributes that support multiple values, the plus (`+`) can be used to list all values that an attribute must have. For example, to indicate that ATTR1 must either be A or (B AND M), use the following: `C[ATTR1]=A,B+M`. Operators may be used too (see table with operators below). This parameter can be used in addition, or instead of, the `key` path parameter. This parameter may be used multiple times (e.g. c[FREQ]=A,M&c[CONF_STATUS]=F).
```

Comment:
Could I still suggest please: 
Line 29: "_All **path** parameters support multiple values, using comma (`,`) as separator._"
Line 30: "_All **path** parameters offer the option to retrieve all existing values, using the `*` operator._"
Line 41: "C" --> "c"

And for TF3, I would need to know the answer to this previous question: "_If the 'context' is wildcarded (or has multiple selections) and the same data are defined for DSD, Dataflow and PA, would the data then have to be returned multiple times, each time in its context? If not, which context should be used by default or in preference?_"

Thanks a lot.

Reply:
Thanks, @dosse, I have made the requested changes.